### DISCUSSION TITLE
Gitub Server side hook

### DISCUSSION THREAD

**Discussion by Aditya-min** [community/community#37606] [Code Security]
Is there way to run an script that locks an developer branch both in lock and remote repository..my use case is i have run the script in CI and CD pipeline once after that that particular branch have to locked then none of them have to make changes in local and remote repository..if anyone knows please tell me and thanks in advance..

**Comment by ValanGracious** [comment]
Same Issue here. Does anyone know?????

**Comment by ossanna16** [comment]
Hi there @Aditya-min and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
